### PR TITLE
Add polaroid to docs/data/projects.yml

### PR DESCRIPTION
**Pull request by Daggy1234**
There was only a single setuptools rust example, that used travisci.

This is an example for github actions with multiple arches, auto release, pypi push and more!

### CODE REVIEW

File: docs/data/projects.yml [hunk 1]
```diff
@@ -293,3 +293,8 @@
   ci: [github]
   os: [apple, linux, windows]
   notes: Time mocking library using only the CPython C API.
+- name: polaroid
+  gh: daggy1234/polaroid
+  ci: [github]
+  os: [apple, linux, windows]
+  notes: Full range of wheels for setuptools rust, with auto release and pypyi deploy.
```
Comment (YannickJadoul) [inline docs/data/projects.yml:300]: Minor typo?

```suggestion
  notes: Full range of wheels for setuptools rust, with auto release and PyPI deploy.
```

**Comment by Daggy1234** [comment]
Yup Good catch!

**Comment by YannickJadoul** [comment]
Thanks!

I ran @henryiii script (first time, but ran smoothly; nice job, @henryiii!) to update `README.md` based on the new `docs/data/project.yml`, but I don't seem to be allowed to push it to your branch.

**Comment by Daggy1234** [comment]
Let me check, I think I had

**Comment by YannickJadoul** [comment]
> Let me check, I think I had
> 
> image

Huh. Wait. It's trying to push to `git@github.com:nsoranzo/cibuildwheel.git`. I'll figure out what's wrong after lunch!

**Comment by YannickJadoul** [comment]
Ha, that seems to have worked. Seems `git` was confused because of an old `patch-1` branch.

Thanks, @Daggy1234!

**Review by YannickJadoul — APPROVED** [review]

**Review by henryiii — APPROVED** [review]

**Pull request merged**